package.json

```diff
@@ -43,6 +43,7 @@
   "devDependencies": {
     "chai": "3.5.0",
     "coffee-19-jshint": "2.3.1",
+    "coffee-jshint": "0.2.3",
```

Comment:
It seems that we have droped coffeescript 1.9 support, so I suggest we remove coffee-19-jshint to stuck to coffee-jshint official release.